### PR TITLE
Fix for certificate issue

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

File: Tasks/Common/azure-arm-rest/AzureServiceClient.ts [hunk 1]
```diff
@@ -122,12 +122,26 @@ export class ServiceClient {
...
+                token = await this.credentials.getToken(true);
+                request.headers["Authorization"] = "Bearer " + token;
+                httpResponse = await webClient.sendRequest(request);
+            }
+        } catch(exception) {
+            let exceptionString: string = exception.toString();
+            if(exceptionString.indexOf("Hostname/IP doesn't match certificates's altnames") != -1
+                || exceptionString.indexOf("unable to verify the first certificate") != -1
+                || exceptionString.indexOf("unable to get local issuer certificate") != -1) {
+                    tl.warning(tl.loc('ASE_SSLIssueRecommendation'));
```
Comment (vincentdass) [inline Tasks/Common/azure-arm-rest/AzureServiceClient.ts]: please add the loc. string to AzureResourceGroupDeployment task.

Comment (kmkumaran) [reply Tasks/Common/azure-arm-rest/AzureServiceClient.ts]: @asranja is moving Webapp V3 to master which will use the same common. I presume we will have to make sure the loc string is present in the V3 task.json as well then? This has to be taken care when you both merge. Please work with @asranja

Comment (Ajay-MS) [reply Tasks/Common/azure-arm-rest/AzureServiceClient.ts]: @vincentdass  common module has localized string. Do we need it in task too?  I think defining the localized string in each task for the common module would not be the right approach.

Comment (vincentdass) [reply Tasks/Common/azure-arm-rest/AzureServiceClient.ts]: discussed offline

**Pull request merged**